iocBoot/iocEK9KTest/gen-cmd.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+"""
+Generate st.cmd from a list of terminals, or the coupler's modbus configuration
+"""
+
+import os
+import sys
+import re
+import argparse
+
+use_modbus = True
+try:
+	from pymodbus.client import ModbusTcpClient
+except:
```

Comment:
```suggestion
except ImportError:
```
Bare `except:` is always recommended against (it can catch things like `OutOfMemoryError` and other weird things you wouldn't want it to). Optional/missing dependency checks should just be catching `ImportError`, typically.